src/data_type/date.js

```diff
@@ -58,7 +58,7 @@ class YotiDate extends Date {
    */
   getMicrosecondTime() {
     const hours = formatDatePart(this.getUTCHours(), 2);
-    const minutes = formatDatePart(this.getUTCMinutes());
+    const minutes = formatDatePart(this.getUTCMinutes(), 2);
     const secondsMicroseconds = formatSecondsWithMicroseconds(
       this.getUTCSeconds(),
       this.getMicroseconds()
```

Comment:
Do we need to add formatting and tests for seconds and microseconds too?

Reply:
I can turn this test into one that will cover all of these.  The test would then be something like `should return zero padded string for hours, minutes and seconds`

Reply:
Fixed in https://github.com/getyoti/yoti-node-sdk/pull/107/commits/72f4c05f9ff036e21dd279e68576f62ba371d334